src/events/VoiceState.ts

```
@@ -25,11 +27,14 @@ export class Voice extends EventHandler<Events.VoiceStateUpdate> {
                 await channel.delete("nobody is in channel")
                 try {
                     await database.deleteChannel(oldId)
-                } catch {
-                    console.log("channel doesnt exist")
+                } catch (error) {
+                    if (error instanceof Prisma.PrismaClientKnownRequestError && error.code == "P2025") return
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)